src/phoenix/db/database.py

```diff
@@ -0,0 +1,351 @@
+import json
+import sqlite3
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Iterator, Optional, Tuple, cast
+
+import numpy as np
+from openinference.semconv.trace import SpanAttributes
+
+from phoenix.trace.schemas import ComputedValues, Span, SpanContext, SpanKind, SpanStatusCode
+
+_CONFIG = """
+PRAGMA foreign_keys = ON;
+PRAGMA journal_mode = WAL;
+PRAGMA synchronous = OFF;
+PRAGMA cache_size = -32000;
+PRAGMA busy_timeout = 10000;
+"""
```

Comment:
note to self: need to figure out how this configuration is settable in something like sql alchemy

Reply:
see https://docs.sqlalchemy.org/en/20/dialects/sqlite.html#foreign-key-support